libs/providers/ofrep-web/src/lib/ofrep-web-provider.spec.ts

```diff
@@ -0,0 +1,176 @@
+import { OfrepWebProvider } from './ofrep-web-provider';
+import TestLogger from './test-logger';
+// eslint-disable-next-line @nx/enforce-module-boundaries
```

Comment:
Can we update the nx boundary so we don't need to disable this rule?

Reply:
Unfortunately, I don't know much about nx, not sure how to disable this rule.

Reply:
I _think_ we can just update this import to use `@openfeature/ofrep-core`. This way we let nx do all the work. All internal imports from other packages in this repo should work this way.

Reply:
Oh I didn't realize this was just testing... I just saw the cross package import....

If this is testing, forget it! I think it's OK.

Reply:
We would probably have to create a test-specific library in NX to do this properly and it's likely not worth the effort.